src/interpolations.jl

```diff
@@ -702,3 +702,8 @@ function value(ip::CrouzeixRaviart{2,1}, i::Int, ξ::Vec{2})
     i == 3 && return 1.0 - 2*ξ_y
     throw(ArgumentError("no shape function $i for interpolation $ip"))
 end
+
+#-------------------------------------------------------------------------------
+# tentative to implement Lagrange element of order k ≥ 3
+include("extend_interpolations.jl")
```

Comment:
I suggest you just add the content of this file directly after the current `Lagrange{2,RefTetrahedron}` in this file instead.